src/consts.js

```diff
@@ -233,6 +233,29 @@ export const ACTOR_LIMITS = {
     LOG_MAX_CHARS: 5000000,
 };
 
+/**
+ * Contains various limits of the Apify platform.
+ */
+export const PLATFORM_LIMITS = {
```

Comment:
These are just defaults, so this should be called DEFAULT_PLATFORM_LIMITS, to make it clear

Reply:
Also, I assume we'll then use these constants in the app, right?

Reply:
Cool, thanks! 
Also, yep, @mtrunkat asked to add this